src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -407,10 +418,39 @@ void CheckIsValidBranchTarget(BasicBlock src, BasicBlock target)
                     else
                         VerificationError(VerifierError.BranchOutOfHandler);
                 }
-                return;
             }
         }
 
+        bool IsDirectChildRegion(BasicBlock enclosingBlock, BasicBlock enclosedBlock)
```

Comment:
Thanks for factoring it into separate method. It would be nice to have a comment about what the method does.